index.js

```diff
@@ -1,6 +1,6 @@
-exports.getAllTweets = (event, context, callback) => {
-  const GetAllTweets = require('./lib/get-all-tweets');
-  GetAllTweets();
+exports.getAllTweets = (/* event, context, callback */) => {
```

Comment:
i commented out the arguments to fix lint errors
however, I dont know if this will mess up lambda. I guess its convention to leave those three args in for lambda??

Reply:
i believe so, but if they are unused like in our case, i think commenting them out is even better.